If the component Development is requested, it is strongly recommended to also
include the component Interpreter to get expected result.
https://cmake.org/cmake/help/v3.25/module/FindPython3.html
Index: src/backends/CMakeLists.txt
--- src/backends/CMakeLists.txt.orig
+++ src/backends/CMakeLists.txt
@@ -64,7 +64,7 @@ endif(QALCULATE_FOUND)
 set_package_properties(Python3 PROPERTIES TYPE OPTIONAL
     PURPOSE "Backend to use Python with Cantor.")
 
-find_package(Python3 COMPONENTS Development)
+find_package(Python3 COMPONENTS Interpreter Development)
 if(Python3_FOUND)
     add_subdirectory(python)
 endif()
